assets/src/hooks/useNotificationsReducer.ts

```diff
@@ -150,10 +158,14 @@ const showLatestNotificationReducer = (
     case "HIDE_LATEST_NOTIFICATION":
       return false
     case "SET_NOTIFICATIONS": {
-      const isInitialLoad = (action as SetNotificationsAction).payload
-        .isInitialLoad
+      const { isInitialLoad, notifications } = (
+        action as SetNotificationsAction
+      ).payload
+
       if (isInitialLoad) {
-        return true
+        return (
+          true && notifications.length > 0 && notifications[0].state !== "read"
+        )
```

Comment:
@lemald this is a change in logic, but I suspect may be needed since the initial load piece wasn't previously loading. In testing locally, I found it strange that a read notification was popping up on load. I was expecting that only a new notification would popup.

Reply:
Makes sense, might be worth running by product / design real quick but I'm like 90% sure that's the right call.